ci-operator/jobs/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main-periodics.yaml

```diff
@@ -93,7 +95,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.14"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-redhat-chaos-prow-scripts-main-4.14-nightly-krkn-hub-cloud-api-tests
+  name: periodic-ci-redhat-chaos-prow-scripts-main-4.14-nightly-krkn-hub-node-tests
   spec:
```

Comment:
lets add in the slack reporting into all of these periodic jobs. Think we are missing a lot of them

Reply:
On it!

Reply:
Done!